File: as_api/src/main/java/com/example/as/api/util/DateUtil.java
package com.example.as.api.util;

import java.text.SimpleDateFormat;
import java.util.Date;

public class DateUtil {
    private static SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");

    /**
     * 返回当前时间
     *
     * @return
     */
    public static String currentDate() {
        return sdf.format(new Date());
    }
}

File: asproj/common/src/main/java/org/devio/as/proj/common/utils/DateUtil.kt
package org.devio.`as`.proj.common.utils

import java.text.SimpleDateFormat
import java.util.*

object DateUtil {
    private const val MD_FORMAT = "MM-dd"
    private const val DEFAULT_FORMAT = "yyyy-MM-dd HH:mm:ss"
    fun getMDDate(date: Date): String {
        val sdf = SimpleDateFormat(MD_FORMAT, Locale.CHINA)
        return sdf.format(date)
    }

    fun getMDDate(dateString: String): String {
        val sdf = SimpleDateFormat(DEFAULT_FORMAT, Locale.CHINA)
        return getMDDate(sdf.parse(dateString)!!)
    }


}